app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -109,9 +109,8 @@
     @BindView(R.id.license_subtitle) TextView licenseSubtitle;
     @BindView(R.id.please_wait_text_view) TextView pleaseWaitTextView;
 
-    //Right Card
-    @BindView(R.id.right_card) CardView rightCard;
-    @BindView(R.id.right_card_expand_button) ImageView rightCardExpandButton;
+
+    //@BindView(R.id.right_card_expand_button) ImageView rightCardExpandButton;
```

Comment:
Please can you remove the commented code - if we do need it later we can always go back into version control.